.stylelintrc

```diff
@@ -0,0 +1,135 @@
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "color-named": "never",
+    "declaration-no-important": true,
+    "font-family-name-quotes": "always-where-recommended",
+    "font-weight-notation": "numeric",
+    "function-url-quotes": "always",
+    "selector-max-specificity": "0,2,0",
+    "selector-no-id": true,
+    "selector-root-no-composition": true,
+    "string-quotes": "single",
+    "declaration-block-properties-order": [
```

Comment:
__Correction:__ about `only 17 color names that are standartized`, looks like there are so called `extended color codes` [1].

[1] https://www.w3.org/TR/2011/REC-css3-color-20110607/#svg-color